content/dstu2/document-reference.md

```diff
@@ -0,0 +1,69 @@
+---
+title: DocumentReference | FHIR DSTU 2 API
+---
+
+# DocumentReference
+
+* TOC
+{:toc}
+
+## Create
+
+Create and store a new clinical note document. 
```

Comment:
"Create new documents. Only documents that would be classified as notes currently supported." ?